ui/scss/component/_navigation.scss

```diff
@@ -15,20 +15,15 @@
 
 .navigation--placeholder {
   @extend .navigation;
-  padding: 2rem 1.5rem;
+  padding: var(--spacing-quarter) 0rem;
```

Comment:
This can be removed since it isn't in a card anymore. The variable can be removed in `vars.scss` too